### ISSUE TITLE
Update Needed to Support HHP DLC Saves?

### ISSUE THREAD

**Issue by ThatAnarchyDude** [kwsch/NHSE#530]
I see NHSE was updated less than 5 hours ago (and I just updated to this current version of NHSE), but my saves have still cannot load in NHSE & have been reading an error about "incorrect size" since the HHP DLC launched.
So can I just NOT use NHSE because I bought the HHP DLC or is there support coming for this?

**Comment by Starri-Knight** [comment]
I don't think it's related, but that's just my experience. I just downloaded it and loaded and saved my files from my switch just fine. I own and was playing the HHP DLC and it's working as intended.

**Comment by ThatAnarchyDude** [comment]
I tried to open my save...the same as it has always been done...by opening the main.dat.
I'm greeted with **"Save file sizes appear to be incorrect.
Continue ?"**

I hit **"Yes"** and get the same error which always happens after a big update until another new version of NHSE is released.
**"Index was outside the bounds of the array."**
The only option is **"Okay"**, but then...
...nothing happens.

I even did a clean install of NHSE just now, and made a new backup of my save via the latest version of JKSV.
Same errors.

Literally the only thing that has changed since the last time I successfully used NHSE & now was the HHP DLC launched & I played through it...designing 30 vacation homes **(which took HOURS)**, but you're saying that's **NOT** issue when I have done everything exactly the way I'm supposed to & am still having this issue?
Seriously?

**Comment by ThatAnarchyDude** [comment]
Okay, Starri-Knight, let me preface this by saying I did have the Emuiibo running when I saved the game which I backed up via JKSV, and I'm guessing that's a problem?

I restored a save from BEFORE the HHP DLC launched.
I went to the archipelago, built 2 vacation homes (so I could take an official Paradise Planning Portfolio picture to get the new "Photographs" folder in my gamesave folder), saved the game (without Emuiibo running), and backed it up with JKSV.
THIS backup (post-HHP) is loading as it should in NHSE.

Sooo...it appears something went wrong during the saving of the save I backed up multiple times via JKSV afterward, and I'm GUESSING that something was having the Emuiibo running while I saved as that HAS caused the CFW to crash the Switch immediately after saving a few saves before the problem save in question.

Sooo...it appears this problem I was having WAS KINDA my fault, and I would like to apologize for the way I acted.
I have...problems...upstairs, and that doesn't excuse the way I lash out sometimes, but it does explain why I can get so frustrated when I believe I'm doing everything right like literally everyone else, but I'm not getting the same result.
I'm so sorry if I offended you or anyone reading this, and I will be closing this issue with this comment...seeing as the issue is technically non-existent.

I WOULD like to request Poki amount editing be added...if it hasn't been already at the time of this comment being posted.

Thank you for the immense amount of time spent reading this.

Stay shiny.

P.S. I feel like such an idiot right now. lol
